src/locationSource/IpBasedLocationSource.js

```diff
@@ -47,6 +55,9 @@ export default class IpBasedLocationSource {
         let coordinates = {};
 
         if (this._coordinatesFieldMode === "singleField") {
+            // CODE_REVIEW zdecydowanie dziwi mnie użycie tutej tej paczki.
```

Comment:
To działałoby tylko w jednym wypadku: kiedy otrzymywane koordynaty znajdowałyby się w roocie odpowiedzi. Jakakolwiek ścieżka bardziej skomplikowana niż nazwa pola nie działałaby już.